.pytest_cache/v/cache/nodeids

```diff
@@ -0,0 +1,58 @@
+[
+  "tests/core/asyncio/test_asyncio_subscriptions_api.py::test_asyncio_stream_api_updates_subscriptions[client-server]",
+  "tests/core/asyncio/test_asyncio_subscriptions_api.py::test_asyncio_stream_api_updates_subscriptions[server-client]",
```

Comment:
Looks like this missed the `gitignore`.

Reply:
Oops https://github.com/ethereum/lahja/pull/125